my_enumerable.rb

```diff
@@ -88,5 +170,4 @@ def multiply_els(array)
     array.my_inject { |result, item| result * item }
   end
 end
```

Comment:
- [ ] `multiply_els` accepts an array as an argument and multiplies all the elements of the array together using `my_inject`

_Yours is currently throwing an error : `NoMethodError: undefined method `multiply_els' for #<RSpec::ExampleGroups::Enumerables::MultiplyEls:0x00007fa0d09bb9f0>`_
![Screen Shot 2020-10-04 at 9 51 55 PM](https://user-images.githubusercontent.com/48804258/95025675-d34a1300-068b-11eb-8e17-e8f9366e6f79.png)